griptape/tasks/__init__.py

```diff
@@ -28,4 +29,5 @@
     "ImageOutpaintingTask",
     "ImageVariationTask",
     "BaseImageGenerationTask",
+    "PythonTask",
```

Comment:
How do we feel about this name @vasinov @andrewfrench @SavagePencil? Feels very literal 😅

Reply:
`ArbitraryFunctionTask`?

Reply:
is this something that could expand beyond Python? Yes, we can see that. We could always make a BaseArbitraryCodeTask or what-have-you that eventually leads to it. But end of the day we supply a callback with context and it goes and does stuff without involving the LLM. "Does it what it says on the tin." Any suggestions for something less literal that leaves doors open?

Reply:
Pretty inconsequential overall but I'd favor CallableTask

Reply:
Agreed, I think I prefer like CallableTask the most.

Reply:
so it shall be

Reply:
CodeExecutionTask